torch/csrc/jit/script/tree_views.h

```diff
@@ -801,7 +801,7 @@ struct Const : public Expr {
   }
   int64_t asIntegral() const {
     try {
-      return c10::stoll(subtree(0)->stringValue());
+      return c10::stoll(subtree(0)->stringValue(), /*pos=*/0, /*base=*/0);
```

Comment:
Why is base 0 here ?

Reply:
`stoll` doesn't check for a `0x` or `0` prefix if `base` is something other than 0

https://en.cppreference.com/w/cpp/string/basic_string/stol

Reply:
and this doesn't mess up normal string processing "10"

Reply:
No it's fine

> If the value of base is ​0​, the numeric base is auto-detected: if the prefix is 0, the base is octal, if the  prefix is 0x or 0X, the base is hexadecimal, otherwise the base is decimal